src/main/java/routes/Get.java

```diff
@@ -5,8 +5,8 @@
 
 public class Get {
     private RequestSpecification request;
-    public String route = "/employees";
-    public String route1 = "/employee/1";
+    public String route = "/employees/";
+    public String action = "/employee/";
```

Comment:
Let it be 
```
private String route = "/employee";
```

Then, you can modify it during the function execution. For example
```
public Response getEmployee(id) {
    return this.request.get(this.route + "/" + id);
}
```